packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -107,6 +108,37 @@ describe('<XGrid />', () => {
       await sleep(50);
       expect(getActiveCell()).to.equal('1-19');
     });
+
+    it('events comming from a component rendered in a cell', async () => {
```

Comment:
I think that we should host the test in DataGrid, this isn't specific to the premium version.

Reply:
There is no need for async

```suggestion
    it('events comming from a component rendered in a cell', () => {
```

Reply:
A more accurate description

```suggestion
    it('should be able to type in an child input', async () => {
```

Reply:
Done